src/server/routes/plywood/plywood.ts

```diff
@@ -77,6 +76,10 @@ router.post("/", (req: SwivRequest, res: Response) => {
         return null;
       }
 
+      // "native" clusters are not defined, maybe they should be defined as some stub object
```

Comment:
It would be worth to at least evaluate the idea at some point
AFAIR we have a few of similar guards across the codebase